include/linux/lsm_hooks.h

```diff
@@ -1532,6 +1535,25 @@ struct security_hook_heads {
 	#undef LSM_HOOK
 } __randomize_layout;
 
+#define SLOT_COUNT 3
+#define FOR_EACH_HOOK_SLOT(M, ...)			\
+	M(0, __VA_ARGS__)				\
+	M(1, __VA_ARGS__)				\
+	M(2, __VA_ARGS__)
+
+struct static_slot {
+	struct static_key_false *key;
+	struct static_call_key *call_key;
+	void *call_tramp;
+};
```

Comment:
Let's have some comments in here explaining what the structure and its fields do (despite it arguable going again the de facto coding style!)